var/spack/repos/builtin/packages/catalyst/package.py

```diff
@@ -206,12 +212,20 @@ def nvariant_bool(feature):
             '-DVTK_USE_X:BOOL=%s' % nvariant_bool('+osmesa'),
             '-DVTK_USE_OFFSCREEN:BOOL=%s' % variant_bool('+osmesa'),
             '-DVTK_OPENGL_HAS_OSMESA:BOOL=%s' % variant_bool('+osmesa'),
-            '-DPARAVIEW_ENABLE_PYTHON:BOOL=%s' % variant_bool('+python')
+            '-DPARAVIEW_ENABLE_PYTHON:BOOL=%s' %
```

Comment:
No longer needed since it's down below with mpi4py now

Reply:
Oh, yes, sorry about that.